OpenRA.Mods.D2k/Traits/Carryable.cs

```diff
@@ -71,12 +74,8 @@ void RequestTransport(CPos destination, Activity afterLandActivity)
 				.Where(c => !c.Trait.IsBusy && !c.Actor.IsDead && c.Actor.Owner == self.Owner && c.Actor.IsInWorld)
 				.OrderBy(p => (self.Location - p.Actor.Location).LengthSquared);
 
-			foreach (var carrier in carriers)
-			{
-				// Notify the carrier and see if he's willing to transport us..
-				if (carrier.Trait.RequestTransportNotify(self))
-					break; // If true then we're done
-			}
+			// Is any carrier able to transport the actor?
+			carriers.Any(carrier => carrier.Trait.RequestTransportNotify(self));
```

Comment:
This feels like an abuse of Any :s.  Could you add a comment like `// Any will return once it finds a carrier that returns true`?